sencha-workspace/packages/slate-cbl/src/view/student/Dashboard.js

```diff
@@ -0,0 +1,197 @@
+
+    config: {
+        contentArea: null,
+        popover: {
+            pointer: 'none'
+        }
+    },
+
+    competenciesTpl: [
+        '<tpl for="competencies">',
+            '{%var level = 9%}', // TODO: real level
+            '{%var studentCompletion = values.studentCompletions[parent.student.ID] || {}%}',
+            '{%var percent = Math.round(100 * (studentCompletion.demonstrationsCount || 0) / values.totalDemonstrationsRequired)%}',
+            '{%var isAverageLow = studentCompletion.demonstrationsAverage < values.minimumAverage && percent >= 50%}',
+            '<li class="panel cbl-competency-panel cbl-level-{[level]}" data-competency="{ID}">',
+                '<header class="panel-header">',
+                    '<h3 class="header-title">{Descriptor}</h3>',
+                '</header>',
+                
+                '<div class="panel-body">',
+                    '<div class="cbl-progress-meter {[isAverageLow ? "is-average-low" : ""]}">',
+                        '<div class="cbl-progress-bar" style="width:{[percent]}%"></div>',
+                        '<div class="cbl-progress-level">L{[level]}</div>',
+                        '<div class="cbl-progress-percent">{[percent]}%</div>',
+                        '<div class="cbl-progress-average">{[fm.number(studentCompletion.demonstrationsAverage, "0.##")]}</div>',
+                    '</div>',
+
+    				'<div class="explainer">',
+                        '<p>{Statement}</p>',
+//						'<p>Here goes a sentence explaining that your average score of <strong>9.25</strong> is below the skill level needed to progress, <strong>10</strong>. This is a sentence describing what you can do to improve.</p>',
+					'</div>',
+
+                    '<ul class="cbl-skill-meter skills-unloaded"></ul>',
+                '</div>',
+            '</li>',
+        '</tpl>'
+    ],
+    
+    skillsTpl: [
+        '<tpl for=".">',
+            '<li class="cbl-skill">',
+                '<h5 class="cbl-skill-name" data-descriptor="{Descriptor}" data-statement="{Statement}">{Descriptor}</h5>',
+                '<ul class="cbl-skill-demos">',
+                    '<tpl for="this.getDemonstrationBlocks(values)">',
+                        '<li class="cbl-skill-demo <tpl if="!counted">cbl-skill-demo-uncounted</tpl>" data-skill="{SkillID}" <tpl if="DemonstrationID">data-demonstration="{DemonstrationID}"</tpl>>',
+                            '<tpl if="Level==0">M<tpl else>{Level:defaultValue("&nbsp;")}</tpl>',
+                        '</li>',
+                    '</tpl>',
+                '</ul>',
+                '<div class="cbl-skill-description"><p>{Statement}</p></div>',
+//                '<div class="cbl-skill-complete-indicator cbl-level-{parent.level} is-checked">',
+//                    '<svg class="check-mark-image" width="16" height="16">',
+//                        '<polygon class="check-mark" points="13.824,2.043 5.869,9.997 1.975,6.104 0,8.079 5.922,14.001 15.852,4.07"/>',
```

Comment:
I'm not sure if this is the best way to output an SVG repeatedly. Are we using glyph fonts yet?